gammapy/data/event_list.py

```diff
@@ -102,7 +102,23 @@ def read(cls, filename, **kwargs):
         table = Table.read(filename, **kwargs)
         return cls(table=table)
 
-    def write(self, filename, gti=None, overwrite=False, format='gadf'):
+    def to_table_hdu(self, name="EVENTS", format='gadf-dl3'):
```

Comment:
Change to `format="gadf"` here...and maybe remove the possibility to change the extname completely? While I think the name is not a GADF specification it might be good be strict on write and use consistent names when writing with Gammapy. If users really want they can change the EXTNAME later on the `BinTableHDU`...